bin/linja.in

```diff
@@ -463,7 +463,7 @@ def make_build_ninja(args):
 
         print("""rule LEAN""", file=f)
         print("""  depfile = ${DLEAN_FILE}""", file=f)
-        print("""  command = "%s" %s $in -o "${OLEAN_FILE}" """ \
+        print("""  command = "%s" %s -o ${OLEAN_FILE} $in""" \
```

Comment:
Interestingly enough, ninja autoquotes the `$in` but *not* the `${OLEAN_FILE}`...  I have now put the quotes back in, even though I don't think that file names with spaces are useful--you can't import them after all.